splash/resources.py

```diff
@@ -54,7 +55,7 @@ def __init__(self, pool, max_timeout, is_proxy_request=False):
         self.max_timeout = max_timeout
 
     def render_GET(self, request):
-        #log.msg("%s %s %s %s" % (id(request), request.method, request.path, request.args))
+        # log.msg("%s %s %s %s" % (id(request), request.method, request.path, request.args))
```

Comment:
Here we followed a convention: for comments use `# comment`, but for commented out code use `#code`. I don't mind changing it though.